pkgs/tools/networking/twa/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, bash, gawk, curl, netcat }:
+
+stdenv.mkDerivation rec {
+  name = "twa-${version}";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = "twa";
+    rev = "${version}";
```

Comment:
I think this could just be `version`, rather than with the interpolation. I copied this from another package definition, though, so I left it in case it's convention.

Reply:
yes, `rev = version;` is preferred here.